test-app/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "test-app",
+  "version": "2.0.1",
+  "description": "Test app for tracked-built-ins addon",
+  "keywords": [],
+  "repository": "https://github.com/tracked-tools/tracked-built-ins",
+  "license": "MIT",
+  "author": "Chris Hewell Garrett",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build",
+    "lint": "npm-run-all lint:*",
+    "lint:template-lint": "ember-template-lint .",
+    "lint:eslint": "eslint .",
+    "lint:tsc": "tsc --noEmit",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "prepublishOnly": "ember ts:precompile",
+    "postpublish": "ember ts:clean"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.5.0",
+    "@glimmer/component": "^1.1.0",
+    "@glimmer/tracking": "^1.1.0",
+    "@types/ember": "^4.0.0",
+    "@types/ember-qunit": "^5.0.0",
+    "@types/ember-resolver": "^5.0.11",
+    "@types/ember__test-helpers": "^2.6.1",
+    "@types/qunit": "^2.11.3",
+    "@types/rsvp": "^4.0.4",
+    "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.3.1",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^2.2.2",
+    "ember-cli": "~4.2.0",
+    "ember-cli-babel": "^7.26.10",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-htmlbars": "^6.0.0",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-typescript": "^4.1.0",
+    "ember-cli-typescript-blueprints": "^3.0.0",
+    "ember-cli-uglify": "^3.0.0",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-export-application-global": "^2.0.1",
+    "ember-load-initializers": "^2.1.2",
+    "ember-maybe-import-regenerator": "^1.0.0",
+    "ember-qunit": "^5.1.5",
+    "ember-resolver": "^8.0.3",
+    "ember-source": "~4.0.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^3.10.0",
+    "ember-test-helpers": "^0.6.3",
+    "ember-tracked-storage-polyfill": "^1.0.0",
+    "ember-try": "^2.0.0",
+    "eslint": "^8.2.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-ember": "^10.4.1",
+    "eslint-plugin-node": "^11.1.0",
+    "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.4.1",
+    "qunit": "2.17.2",
+    "qunit-dom": "^2.0.0",
+    "release-it": "^14.6.1",
```

Comment:
@linainu let's remove this `release-it` and `release-it-lerna-changelog` from this file, it will be moved to top level per another comment